### DISCUSSION TITLE
Unable to config Starship

### DISCUSSION THREAD

**Discussion by aarabii** [starship/starship#4679] [Q&A]
Everytime after adding `Invoke-Expression (&starship init powershell)` i'm getting this error `System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1+AsyncStateMachineBox`1[System.String,System.IO.StreamReader+<ReadToEndAsyncInternal>d__66]` and I don't know what to do

can someone help me...

**Comment by davidkna** [comment]
Error output won't display correctly in older PowerShell versions until the next starship release. Please try inspecting the log files in `~\.cache\starship` or use a newer PowerShell version to inspect the output.

**Comment by aarabii** [comment]
Hey @davidkna  thanks for your response I'm using PowerShell 7 I fixed that issue using a youtube video which is about customizing the terminal but my terminal doesn't show those effects....
